explainer.md

```diff
@@ -147,13 +152,13 @@ for await (const stream of transport.receiveBidirectionalStreams()) {
 
 ```javascript
 const mediaSource = new MediaSource();
-await new Promise(resolve => mediaSource.addEventListener('sourceopen', () => resolve(), {once: true}));
+await new Promise(resolve => mediaSource.addEventListener('sourceopen', resolve, {once: true}));
 const sourceBuffer = mediaSource.addSourceBuffer('video/webm; codecs="opus, vp09.00.10.08"');
-const transport = new Http3Transport("/video");
-await fetch('http://example.com/babyshark');
-for await (const datagram of transport.receiveDatagrams()) {
+const transport = new WebTransport('/video');
+await fetch('https://example.com/babyshark');
```

Comment:
This is odd. Do you know what this fetch is doing here?

Reply:
I don't know. It looks like a mistake, so I removed it.